tests/test_ess_csc.py

```diff
@@ -247,3 +256,45 @@ async def test_rpi_data_client_cannot_connect(self) -> None:
             await self.assert_next_sample(
                 topic=self.remote.evt_errorCode, errorCode=ErrorCode.ConnectionFailed
             )
+
+    @mock.patch(
+        "lsst.ts.ess.csc.rpi_data_client.COMMUNICATE_TIMEOUT", COMMUNICATE_TIMEOUT
+    )
+    async def test_rpi_data_client_timeout(self) -> None:
+        async with self.make_csc(
+            initial_state=salobj.State.ENABLED,
+            config_dir=TEST_CONFIG_DIR,
+            simulation_mode=1,
+        ):
+            await self.assert_next_summary_state(salobj.State.ENABLED)
+            await self.assert_next_sample(topic=self.remote.evt_errorCode, errorCode=0)
+            assert len(self.csc.data_clients) == 1
+            for data_client in self.csc.data_clients:
+                assert data_client.mock_server.connected
+
+            await self.validate_telemetry()
+
+            # Set the wait time for the device to SHORT_WAIT_TIME, which is a
+            # higher value than the default (see
+            # lsst.ts.ess.common.device.MockDevice).
+            common.device.MockDevice.wait_time = SHORT_WAIT_TIME
```

Comment:
It surprises me to see SHORT_WAIT_TIME being described as a *higher* value than the default. Please consider LONG_WAIT_TIME here and TOO_LONG_WAIT_TIME (or some other variable that clearly indicates that you expect the read to time out).

It might be clearer and _perhaps_ safer to compute the value as a multiple of the standard/default wait time. For example if the standard value is common.device.WAIT_TIME then you could use:
```
standard_wait_time = common.device.WAIT_TIME
common.device.MockDevice.wait_time = standard_wait_time * 2
```
here and a much larger multiple where you use LONG_WAIT_TIME below.

Reply:
Done